azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -549,6 +549,13 @@ public long getQueuedFlowSize() {
     return this.queuedFlows.size();
   }
 
+  @Override
+  public long getAgedQueuedFlowSize() {
+    logger.error("Unsupported Function: getQueuedFlowSize() - Retrieving of Aged flows that are "
```

Comment:
Did you mean get**Aged**QueuedFlowSize() in the message?

Reply:
Yes, changed.